### ISSUE TITLE
Changing 'sender' in signal to be object instead of class

### ISSUE THREAD

**Issue by J-F-Far** [anx-ckreuzberger/django-rest-passwordreset#29]
Hello,

Thanks for your work on this module!

Would it be possible to pass the actual view object as the sender of the signal handlers, rather than simply the class of the view?

Using DjangoRESTFramework, I've encountered a situation where I need to generate an email containing a link to a password reset form that is in a React app hosted alongside my DRF API. There is no request context for me to determine the domain, since the signal 'sender' is only the class, and not the actual object that generated the signal. There is also a design consideration that prevents me from simply hard-coding the domain value in the application settings.

If the signal sender passed was the actual instance(view) that generated the signal, then I could use the .request attribute of the view. 

I think this would be a small/easy/backward compatible change. 

Thanks for your consideration,

Joel

**Comment by anx-ckreuzberger** [comment]
Hi,

I wouldn't want to change `sender` to be an instance, but in the spirit of [Djangos `post_save`/`pre_save`/... signal](https://docs.djangoproject.com/en/2.2/ref/signals/#post-save) I could add `instance=` here: 
https://github.com/anx-ckreuzberger/django-rest-passwordreset/blob/53123f94b42f5ad918f1ec7d339623d3752ce961/django_rest_passwordreset/views.py#L144-L146

```python
                # send a signal that the password token was created
                # let whoever receives this signal handle sending the email for the password reset
                reset_password_token_created.send(sender=self.__class__, instance=self, reset_password_token=token)
```

**Comment by anx-ckreuzberger** [comment]
Released in 1.0.0alpha3
https://github.com/anx-ckreuzberger/django-rest-passwordreset/releases/tag/1.0.0a3

**Comment by J-F-Far** [comment]
Great. This should solve my problem.

Thanks for getting to this so quickly!
